src/NetworkName.php

```diff
@@ -201,23 +202,23 @@ public static function rawSearchOptionsToAdd(array &$tab, array $joinparams)
         ];
 
         $tab[] = [
-            'id'                 => '127',
```

Comment:
Next time, please avoid useless spaces only changes, this just make PR more complex to read ;)